src/Cake.Npm/Prune/NpmPruneSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Cake.Npm.Prune
+{
+    using Core;
+    using Core.IO;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Contains settings used by <see cref="NpmPruneRunner"/>.
+    /// </summary>
+    public class NpmPruneSettings : NpmSettings
+    {
+        private readonly List<string> _arguments = new List<string>();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NpmPruneSettings"/> class.
+        /// </summary>
+        public NpmPruneSettings()
+            : base("prune")
+        {
+        }
+
+        /// <summary>
+        /// Arguments to pass to npm.
+        /// </summary>
+        public IList<string> Arguments => _arguments;
```

Comment:
There's no need to provide a generic Arguments property here. There's already the [ArgumentCustomization](https://cakebuild.net/api/Cake.Core.Tooling/ToolSettings/50AAB3A8) property which can be used to add additional or manipulate arguments.

Arguments provided here should be the specific arguments as supported by npm (e.g package scope/name, production flag). But I would also be OK to take this PR without any specific arguments and add them later if there's a need.